specs/Deployment/DeploymentAPI.md

```diff
@@ -246,12 +264,164 @@ Here's the previous example extended with the `Show UI on error` option:
+repair has failed and Initialize returns with PackageRepairFailed status (OR) it is possible that
+Main package is repaired successfully but the Singleton package repair has failed and Initialize
+returns with repair has failed and Initialize returns with PackageRepairFailed status (OR) it is
+possible that status. In both cases, the returned WindowsAppRuntimeStatus will contain the error of
+first package repair failure. There will be no rollback of any successfully repaired packages. See
+the
+[developer docs](https://docs.microsoft.com/windows/apps/windows-app-sdk/deploy-packaged-apps#address-installation-errors)
+for instructions on how to handle errors.
+
+```C#
+    if (DeploymentManager.GetStatus().Status == DeploymentStatus.PackageRepairRequired ||
+        DeploymentManager.GetStatus().Status == DeploymentStatus.Unknown)
+    {
+        // Repair will always attempt to repair WindowsAppRuntime regardless of it's state.
+        // Repair can sometimes take several seconds to deploy the packages.
+        // These should be run on a separate thread so as not to hang your app while the
+        // packages deploy.
+        var repairTask = Task.Run(() => DeploymentManager.Repair());
+        //...do other work while the repair is running...
+        repairTask.Wait();
+
+        // Check the result.
+        if (repairTask.Result.Status != DeploymentStatus.Ok)
+        {
+            // The Repair() has failed.
+            // Do error reporting or gather information for submitting a bug.
+            // Gracefully exit the program or carry on without using the WindowsAppRuntime.
+        }
+    }
+```
+
+### DeploymentRepairOptions
+
+#### ForceDeployment
+
+When this API may install newer version of one or more of the WinAppSDK packages (i.e. an update)
+and if there are any currently running process(es) associated with these WinAppSDK package(s) (in
+other words, if any of the WinAppSDK packages to be updated are currently in use), then this API
+will fail installing that WinAppSDK package. But this update to the WinAppSDK packages can be forced
+by using DeploymentRepairOptions object and setting ForceDeployment option before passing it to this
+API. This option when set will shut down the application(s) associated with WinAppSDK packages,
+update the WinAppSDK packages and restart the application(s).
+
+If this option is set when updating Singleton package, then the out of process com server from the
+Singleton package such as push Notifications is explicitly restarted.
+
+When the API is updating Framework package and ForceDeployment option is set, then all dependent
+packages that are NOT currently in use will be immediately re-installed to refer to the updated
+Framework package and all dependent packages that are currently in use will be re-installed, after
+they shut down, to refer to the updated Framework package.
+
+```C#
+    if (DeploymentManager.GetStatus().Status == DeploymentStatus.PackageInstallRequired)
```

Comment:
This is in the DeploymentRepairOptions section, so this should be checking for == DeploymentStatus.PackageRepairRequired right?

Reply:
Yes. Need to check for  == DeploymentStatus.PackageRepairRequired